cmd/restic/cmd_check.go

```diff
@@ -479,3 +506,15 @@ func selectRandomPacksByFileSize(allPacks map[restic.ID]int64, subsetSize int64,
 	packs := selectRandomPacksByPercentage(allPacks, subsetPercentage)
 	return packs
 }
+
+type checkResult struct {
+	Error           bool     `json:"error"`
+	NumIndexErrors  int      `json:"num_index_errors"`
+	NumPackErrors   int      `json:"num_pack_errors"`
+	NumTreeErrors   int      `json:"num_tree_errors"`
+	NumUnusedBlobs  int      `json:"num_unused_blobs"`
+	NumDataErrors   int      `json:"num_data_errors"`
+	BrokenPacks     []string `json:"broken_packs"`         // run "restic repair packs ID..." and "restic repair snapshots --forget" to remove damaged files
```

Comment:
Do we want these IDs in the status or does this encourage people to script such destructive commands?